Gen5Dashboard/Gen5Dashboard.cpp

```diff
@@ -12,10 +12,31 @@ Gen5Dashboard::Gen5Dashboard(int& argc, char** argv)
     , communicationContainer_(new CommunicationContainer())
     , businessContainer_(new BusinessContainer(*communicationContainer_, *dataContainer_))
     , presenterContainer_(new PresenterContainer(*dataContainer_))
-    , viewContainer_(new ViewContainer(*presenterContainer_))
 {
+    char mode = parseCommandLineArgs(argc, argv);
+    viewContainer_.reset(new ViewContainer(*presenterContainer_, mode));
 }
 
 Gen5Dashboard::~Gen5Dashboard()
 {
 }
+
+char Gen5Dashboard::parseCommandLineArgs(int& argc, char** argv)
+{
+    //This function returns 'd' for display mode and 'r' for race mode
+    //The default mode is display mode
+    if (argc == 2)
+    {
+        if (QString(argv[1]) == "-d" || QString(argv[1]) == "-D")
```

Comment:
Qt has it's own classes that are used for command line arguments such as QCommandLineParser and QCommandLineOption. These classes are a bit more cleaner and have more options, so I think we should use them.
http://doc.qt.io/qt-5/qcommandlineparser.html
http://doc.qt.io/qt-5/qcommandlineoption.html

An example of their use can be found in `Recruit-Resources/Advanced-Recruit-Training/src/main.cpp`.